Assets/Scripts/ResourceManager.cs

```diff
@@ -9,7 +9,7 @@ public class ResourceManager : MonoBehaviour
 {
     public float initialFuel;
     private float fuelRemaining, massCollected, timeElapsedSec, lastHitTimeSec;
-    public float timeBetweenHits = 5.0f;
```

Comment:
I found out this is actually what solved the bug: in at least one level, this value was set to 15 seconds, so there would be a pretty long period of time where you could not make the next golf hit. I just made this private because I don't think we want it to have different values for different levels.

Reply:
thanks! nice work